cpp/test/Common/TestHelper.cpp

```diff
@@ -200,7 +200,7 @@ Test::TestHelper::getTestEndpoint(const Ice::PropertiesPtr& properties, int num,
     std::string protocol = prot;
     if (protocol.empty())
     {
-        protocol = properties->getPropertyWithDefault("Ice.Default.Protocol", "default");
```

Comment:
This `getPropertyWithDefault` did not make sense. 

transport `default` means "use the transport onfigured by Ice.Default.Protocol, and the default Ice.Default.Protocol is tcp. Setting Ice.Default.Protocol to 'default' should be disallowed since it's circular.